src/LibraryViewExtensionWebView2/LibraryViewController.cs

```diff
@@ -197,7 +198,8 @@ private string ReplaceUrlWithBase64Image(string html, string minifiedURL, bool m
                 base64 = $"data:application/x-font-{ext};charset=utf-8;base64,{base64}";
             }
 
-            html = html.Replace(minifiedURL, '"' + base64 + '"');
+            //In Libraryjs project, Webpack5 is removing the initial slash "/" when loading svg files so in the html we have the string "resources/image.svg" then also we need to remove the slash (the first char) otherwise it won't be replaced by the base64 content.
```

Comment:
is this problem only with loading svgs or with loading any image format?

Reply:
The problem is for the resources in the image, I updated the comment. commit:https://github.com/DynamoDS/Dynamo/pull/13417/commits/2734ad37801ff641ecaf4aa5802a65091ab4b286
 
![image](https://user-images.githubusercontent.com/61755417/197284670-b0fd58d6-65f1-45a0-a033-bdea2a79393a.png)